### ISSUE TITLE
calculate the resonance position in "Calculate_fields"

### ISSUE THREAD

**Issue by nilsmelchert** [Jerome-LMA/oscar#8]
I was wondering, if we should directly calculate the resonance position instead of calling an error as done here:

https://github.com/Jerome-LMA/oscar/blob/12addde41351e1fc6c62873801b8fadbb21a4b8b/Classes/%40Cavity1/Calculate_fields.m#L21

and here:

https://github.com/Jerome-LMA/oscar/blob/12addde41351e1fc6c62873801b8fadbb21a4b8b/Classes/%40Cavity1/Calculate_fields_AC.m#L21

Any thoughts on this?

**Comment by Jerome-LMA** [comment]
Yes it should, if the resonance phase is not present calculate it.
